pact-data-model/src/lib.rs

```diff
@@ -361,9 +357,18 @@ pub struct ProductOrSectorSpecificRuleSet(pub Vec<ProductOrSectorSpecificRule>);
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq, JsonSchema)]
 pub struct CrossSectoralStandardSet(pub Vec<DeprecatedCrossSectoralStandard>);
 
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
+// TODO: use Enum instead
+pub struct ISO3166CC(pub Alpha2CountryCode);
```

Comment:
`ISO3166CC` data type now looks redundant to me (or `Alpha2CountryCode`). Would you please explain why you introduce the later data type / why you are wrapping it in `ISO3166CC` as well?

Reply:
Fair point! I was trying to change as little as possible and `ISO3166CC` use to wrap a `String`, so I just kept that structure. But it is indeed redundant. I'll update it accordingly!